docs/application/native/guides/app-management/sticker.md

```diff
@@ -0,0 +1,431 @@
+# Sticker
+
+
+The Sticker feature provides the functions to manage and retrieve sticker information. The Sticker application is divided into the [Sticker provider application](#provider_application) that creates the sticker and the [Sticker consumer application](#consumer_application) that uses the sticker created by the Sticker provider. The Sticker provider application can store sticker details such as URI, keyword, and group name to the Sticker database. The Sticker consumer application can retrieve sticker information using the Sticker API.
+
+The main features of the Sticker API include:
+
+- Preparing the Sticker service for use
+
+  You can connect the Sticker daemon to be able to operate the Sticker service.
+
+- Sticker data handle
+
+  You can use the Sticker data handle to [store the sticker information](#set_data_handle) and to [get the sticker information](#get_data_handle) stored in it.
+
+- [Inserting the sticker information](#insert_sticker)
+
+  - You can store sticker information using the Sticker API at application runtime, and [use JSON file](#set_json) to store many sticker information at once.
+  - You can store sticker information automatically when the Sticker provider application is installed without using the Sticker API.
+
+- Retrieving sticker information
+
+  You can [retrieve sticker information](#retrieve_sticker), which includes URI, keyword, group name and so on.
+
+**Figure: Sticker mechanism**
+
+![Sticker mechanism](./media/sticker.png)
+
+<a name="provider_application"></a>
+## Creating Sticker provider application
+
+You can provide the sticker information to applications that use sticker. If you store the stickers into the Sticker database, the Sticker consumer applications can search for them.
+
+### Prerequisites
+
+To enable your Sticker provider application to use the Sticker functionality:
+
+1. To use the Sticker provider API, the following category must be added to the `tizen-manifest.xml` file:
+
+    ```
+    <category name="http://tizen.org/category/sticker"/>
+    ```
+
+2. To use the functions and data types of the Sticker API, include the `<sticker_provider.h>` header file in your application:
+
+    ```
+    #include <sticker_provider.h>
+    ```
+
+ To ensure that a Sticker function has been executed properly, make sure that the return is equal to `STICKER_ERROR_NONE`.
```

Comment:
The intention here is to avoid the usage of "make sure".
However, in many other guides, "make sure" is used. 
To maintain consistency with other guides, please ignore my comment or change to "ensure that".
@mijin85cho 
Request your input.

Reply:
Like other guides, I will use 'make sure'.